source/docs/software/wpilib-tools/robot-characterization/characterization-routine.rst

```diff
@@ -53,6 +53,15 @@ Follow the instructions in the pop-up windows after pressing each test button.
 .. image:: images/running-the-test.png
    :alt: Running the various tests
 
+Trackwidth
+^^^^^^^^^^
+
+The Trackwidth test will spin your robot in an attempt to measure the distance between your wheels.
+
+.. note:: If your robot is having trouble turning during the Trackwidth test you should increase the :guilabel:`Rotation Wheel voltage (V):` value and run the test again.
+
+.. note:: Trackwidth values for high friction wheels (like pneumatic tires) may be measured incorrectly and a measured value should be used.
```

Comment:
This is misleading. The track width value that frc-char will calculate for drivetrains with high-friction wheels *is* the value that should be used everywhere in code -- it just won't match with the physical reality.

The physical measurement should *not* be used because it won't accurately represent the angular dynamics.

Reply:
That isn't what I understood from @calcmogul.   He also said don't do it with pneumatics but that is probably for a different note...maybe?

Reply:
What Prateek said. And yeah, avoid pneumatic wheels because the encoders aren't as accurate, even with a correct empirical trackwidth.

Reply:
So what do you think this note should say something like:
"Trackwidth values for high friction wheels (like pneumatic tires) may be significantly different from the measured value."